Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -45,6 +45,34 @@ public IActionResult Book(string isbn)
             }
             return View(new BookViewModel(book));
         }
+        public IActionResult AddBook(string errorMessage = "")
```

Comment:
The get endpoint shouldn't take the error message as an argument, because then the user can change the url to display any error message they want. Instead you should construct the error message in the view model. All you need to pass to the view model is some indication of whether the isbn is valid, and the arguments the user had typed in. You should show the old arguments with the error, otherwise the user might get confused 🙂 Does that make sense? 🙂